infrastructure/main.tf

```diff
@@ -110,3 +110,56 @@ resource "azurerm_key_vault_secret" "draft-store-key" {
   value        = random_string.draft_encryption_key.result
   key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
 }
+
+////////////////////////////////
+// DB version 11             //
+////////////////////////////////
+
+module "data-store-db-v11" {
+  source          = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product         = "${var.component}-db-v11"
+  component       = var.component
+  name            = "${local.app_full_name}-postgres-db-v11"
+  location        = "${var.location}"
+  env             = "${var.env}"
+  subscription    = "${var.subscription}"
+  postgresql_user = "${var.postgresql_user}"
+  database_name   = "${var.database_name}"
+  postgresql_version = "11"
+  sku_name        = "${var.database_sku_name}"
+  sku_tier        = "GeneralPurpose"
+  sku_capacity    = "${var.database_sku_capacity}"
+  storage_mb      = "${var.database_storage_mb}"
+  common_tags     = "${var.common_tags}"
+}
+
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
+  name         = "${var.component}-POSTGRES-USER-V11"
```

Comment:
do we need to do phased update?? I mean create V11 database first and then migrate data and then pointing application variables to new database..?